server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -165,6 +165,7 @@ static TransportVersion def(int id) {
     public static final TransportVersion REPO_ANALYSIS_REGISTER_OP_COUNT_ADDED = def(8_534_00_0);
     public static final TransportVersion ML_TRAINED_MODEL_PREFIX_STRINGS_ADDED = def(8_535_00_0);
     public static final TransportVersion COUNTED_KEYWORD_ADDED = def(8_536_00_0);
+    public static final TransportVersion VECTOR_OPS_COUNT_ADDED = def(8_537_00_0);
```

Comment:
TransportVersion in the main [branch](https://github.com/elastic/elasticsearch/blob/f89f82a8eeb4e30e24ec9025864aacc1a4e9a40b/server/src/main/java/org/elasticsearch/TransportVersions.java#L171) is

`public static final TransportVersion ML_STATE_CHANGE_TIMESTAMPS = def(8_540_00_0);`

In the **lucene_snapshot** branch, should we make the latest to 8_541_00_0?

@benwtrent, @kderusso

Reply:
@saikatsarkar056 main will be merged into lucene-snapshot and probably cause a conflict here. 

Then when we cut the Lucene release to merge into main (from lucene_snapshot) we will likely have to update all these transport changes. Its ok.